scania-digital-design-system/tegel
# feat: add tdsClick event to table rows

## **Describe pull-request**  
This pull-request is regarding adding an tdsClick event to the table rows handle events on a row click.

## **Issue Linking:**  
_Choose one of the following options_
- **GitHub:** [Include issue link  ](https://github.com/scania-digital-design-system/tegel/issues/672)
- **No issue:** Adding a tdsClick event to the table rows

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to storybook and navigate to the table component > basic row.
2. Check in the console that the event is being triggered upon clicking a row.

## **Checklist before submission**

## **Suggested test steps**
